pkg/server/chaosd/disk.go

```diff
@@ -149,12 +151,37 @@ func (s *Server) DiskFill(fill *core.DiskCommand) (uid string, err error) {
 		}()
 	}
 
+	if fill.Size != "" {
+		fill.Size = strings.Trim(fill.Size, " ")
+	} else if fill.Percent != "" {
+		fill.Percent = strings.Trim(fill.Percent, " ")
+		percent, err := strconv.ParseUint(fill.Percent, 10, 0)
+		if err != nil {
+			log.Error(fmt.Sprintf("err when parsing disk percent '%s'", fill.Percent), zap.Error(err))
+			return uid, err
+		}
+		dir, err := os.Getwd()
+		if err != nil {
+			log.Error(fmt.Sprintf("unexpected err when using os.Getwd"), zap.Error(err))
+			return uid, err
+		}
+		di, err := disk.GetInfo(dir)
+		if err != nil {
+			log.Error(fmt.Sprintf("unexpected err when using disk.GetInfo"), zap.Error(err))
+			return uid, err
+		}
+		totalM := di.Total / 1024 / 1024
+		fill.Size = strconv.FormatUint(totalM*percent/100, 10)
+	} else {
+		fill.Size = "0"
+	}
```

Comment:
how about do some validate work in `func (d *DiskCommand) Validate() error`, for example, at least one of `Size` and `Percent` can not be empty.

Reply:
I agree with you.